var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -33,8 +33,7 @@ class Nwchem(Package):
     homepage = "http://www.nwchem-sw.org"
     url      = "http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz"
 
-    version('6.6', 'c581001c004ea5e5dfacb783385825e3',
-            url='http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz')
+    version('6.6.revision27746-src.2015-10-20', 'c581001c004ea5e5dfacb783385825e3')
```

Comment:
i checked http://www.nwchem-sw.org/index.php/Download (for previous release) as well as their forum http://www.nwchem-sw.org/index.php/Special:AWCforum/sf/id4/Compiling_NWChem.html// and looks like `6.6` should be enough.